src/Entity/ControllerPageInterface.php

```diff
@@ -14,7 +14,7 @@ interface ControllerPageInterface
     /**
      * Returns a controller reference (string) which is responsible for rendering the page.
      *
-     * @return mixed
+     * @return string|null
```

Comment:
Ik had aangegeven dit ook gelijk aan te passen. Een type in een interface veranderen zou in theorie breaking zijn, echter is dit een annotation en zal PHP daar zelf niks mee doen. Het moet sowieso wel een string zijn (of null bij deze), want de `forward()` method accepteert ook alleen maar een string.

Reply:
In PGO zie ik dat het niet altijd een 'class-string' is, maar ook `return OverviewController::class . '::contentSearchAction';`.  Dus ik denk dat het `string|null` moet blijven dan...

https://github.com/zicht/pgosupport.nl/pull/596/files#diff-12fa57b6dcfb762fa6dc2f0902bfac3bfc039796450870adeb3c45093f1337adR95

Reply:
Klopt, of `string|class-string|null`, maar daar heb je dan weer niet zoveel aan gok ik 😄

Reply:
> Klopt, of `string|class-string|null`, maar daar heb je dan weer niet zoveel aan gok ik

Niet zoveel aan idd, dus `string|null` is dan het beste.